ConsentViewController/Classes/SourcePointClient/MessagesResponse.swift

```diff
@@ -26,17 +26,10 @@ enum MessageCategory: Int, Codable, Defaultable, Equatable {
 
     var campaignType: SPCampaignType {
         switch self {
-        case .gdpr:
-            return .gdpr
-
-        case .ccpa:
-            return .ccpa
-
-        case .ios14:
-            return .ios14
-
-        default:
-            return .unknown
+        case .gdpr: .gdpr
+        case .ccpa: .ccpa
+        case .ios14: .ios14
+        default: .unknown
```

Comment:
No changed logic, simply removing unnecessary `return` keyword and inlining things.

Reply:
I don't know swift, but can you omit the `return` keyword when you have to return a certain value?

Reply:
@bohdan-go-wombat yes, if that's the only statement in a function. A switch statement _kinda_ returns a value after the `case :`, that's why it works in this case.

Reply:
@bohdan-go-wombat it turns out older versions of Swift complained about that 😅 , reverted back to have the `return` keyword.